spring-boot-project/spring-boot-docs/src/main/asciidoc/using-spring-boot.adoc

```diff
@@ -979,21 +981,21 @@ Then you need to set a `spring.devtools.remote.secret` property, for example:
 WARNING: Enabling `spring-boot-devtools` on a remote application is a security risk. You
 should never enable support on a production deployment.
 
-Remote devtools support is provided in two parts; there is a server side endpoint that
-accepts connections, and a client application that you run in your IDE. The server
+Remote devtools support is provided in two parts: a server_side endpoint that
+accepts connections and a client application that you run in your IDE. The server
 component is automatically enabled when the `spring.devtools.remote.secret` property
 is set. The client component must be launched manually.
 
 
 
-==== Running the remote client application
+==== Running the Remote Client Application
 The remote client application is designed to be run from within your IDE. You need to run
-`org.springframework.boot.devtools.RemoteSpringApplication` using the same classpath as
-the remote project that you're connecting to. The _non-option_ argument passed to the
```

Comment:
I think non-option was right here. It's the difference between `whatever` and `--whatever`. Perhaps there's a better way to describe that?

Reply:
I changed the sentence to describe the single required argument. It now reads, "The application's single required argument is the remote URL to which it connects."

Reply:
👍